build.gradle

```diff
@@ -359,12 +359,8 @@ task generateJournalAbbreviationList(type: JournalAbbreviationConverter) {
 apollo {
   packageName = "org.jabref.jabrefonline"
   schemaFile = file("src-gen/main/java/org/jabref/jabrefonline/graphql/schema.json")
-  customScalarsMapping = [
-    "DateTime" : "java.time.LocalDateTime"
-  ]
+  mapScalar("DateTime", "java.time.LocalDateTime", "org.jabref.logic.jabrefonline.JavaLocalDateTimeAdapter.INSTANCE")
```

Comment:
If you use Z you probably need to have a ZonedDateTime object

Reply:
Thanks, good point! Fixed